android/src/main/java/com/aerobotics/DjiMobile/DJIMobile.java

```diff
@@ -511,7 +511,7 @@ public void stopRecordRealTimeData() {
     }
 
     @ReactMethod
-    public void getLogPath(Promise promise) {
+    public void getFlightLogPath(Promise promise) {
```

Comment:
What were your thoughts on renaming this? Is it not more consistent to name these the same as the DJI SDK method names especially if the wrapper could be used by other people in different projects down the line?